packages/common/src/content/fetch.ts

```diff
@@ -251,13 +251,16 @@ export const fetchHubContent = async (
   requestOptions: IRequestOptions
 ): Promise<IHubEditableContent> => {
   // NOTE: b/c we have to support slugs we use fetchContent() to get the item
-  // by telling it to not fetch any enrichments
-  // which we then fetch as needed after we have the item
+  // by telling it to not fetch any enrichments which we then fetch as needed after we have the item
+  // EXCEPTION: We ask it to fetch the metadata enrichment so we don't have to parse xml on the client
   const options = {
     ...requestOptions,
-    enrichments: [],
+    enrichments: ["metadata"],
   } as IFetchContentOptions;
-  const { item, access } = await fetchContent(identifier, options);
+  const { item, access, additionalResources } = await fetchContent(
```

Comment:
so `additionalResources` is only available from the v3 API? If not, let's fetch it like any other enrichment below.

Reply:
If we are good parsing the XML on the client, then we can absolutely fetch it below and not rely on the v3 api. We already have util functions to convert metadata JSON into the additional resources.